core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5201,7 +5201,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String WORKER_KEYTAB_FILE = "alluxio.worker.keytab.file";
     public static final String WORKER_MASTER_CONNECT_RETRY_TIMEOUT =
         "alluxio.worker.master.connect.retry.timeout";
-    public static final String WORKER_MEMORY_SIZE = "alluxio.worker.memory.size";
```

Comment:
Update alluxio.conf.RemovedKey#REMOVED_KEYS

Reply:
is it still a removed key if we have an alias for it?

Reply:
yes we still put them in `RemovedKey`, just check it out and you will find examples